libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/ServerPickerActivity.java

```diff
@@ -244,4 +243,27 @@ public void rebuildDisplay() {
             }
         }
     }
+
+    private class AuthConfigTask extends AsyncTask<Void, Void, Void> {
```

Comment:
<table data-meta="generated_by_SalesforceSDK">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/13a92a93ed6f93479f8ed1b774cb2f4d10c7e044/libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/ServerPickerActivity.java#L247"></span>This AsyncTask class should be static or leaks might occur (com.salesforce.androidsdk.ui.ServerPickerActivity.AuthConfigTask)</td>
    </tr>
  </tbody>
</table>